tasks.rb

```diff
@@ -2424,6 +2424,20 @@ def create_sample_hpxmls
                   'base-hvac-air-to-air-heat-pump-1-speed-seer2-hspf2.xml',
                   'base-hvac-air-to-air-heat-pump-2-speed.xml',
                   'base-hvac-air-to-air-heat-pump-var-speed.xml',
+                  'base-hvac-air-to-air-heat-pump-var-speed-detailed-performance.xml',
+                  'base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-heating-only.xml',
+                  'base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-normalized-capacities.xml',
+                  'base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml',
+                  'base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-other-temperatures.xml',
+                  'base-hvac-central-ac-only-var-speed-detailed-performance.xml',
+                  'base-hvac-central-ac-only-var-speed-detailed-performance-autosize.xml',
+                  'base-hvac-install-quality-air-to-air-heat-pump-var-speed-detailed-performance.xml',
+                  'base-hvac-mini-split-air-conditioner-only-ductless-detailed-performance.xml',
+                  'base-hvac-mini-split-air-conditioner-only-ductless-detailed-performance-autosize.xml',
+                  'base-hvac-mini-split-heat-pump-ducted-detailed-performance.xml',
+                  'base-hvac-mini-split-heat-pump-ducted-detailed-performance-autosize.xml',
+                  'base-hvac-mini-split-heat-pump-ductless-detailed-performance.xml',
+                  'base-hvac-mini-split-heat-pump-ductless-detailed-performance-autosize.xml',
```

Comment:
These files didn't get added to the repo.

Also, this is a lot of test files to add, which will slow down the CI. I suggest paring it down to the minimum set of files that provide adequate testing. All you're doing here is testing that OS-ERI correctly handles the inputs; the underlying functionality is already tested in OS-HPXML.